src/Tutor.Infrastructure/Database/TutorContext.cs

```diff
@@ -191,6 +191,8 @@ private static void ConfigureKnowledgeMastery(ModelBuilder modelBuilder)
         {
             trackerBuilder.Property(tracker => tracker.CountOfSessions).IsRequired().HasDefaultValue(0);
             trackerBuilder.Property(tracker => tracker.DurationOfFinishedSessions).IsRequired().HasDefaultValue(TimeSpan.Zero);
+            trackerBuilder.Property(tracker => tracker.IsPaused).IsRequired().HasDefaultValue(false);
+            trackerBuilder.Property(tracker => tracker.DurationOfFinishedPauses).IsRequired().HasDefaultValue(TimeSpan.Zero);
```

Comment:
Here we set default to TimeSpan.Zero in DB, but in the SessionTracker line 17 we set it to null. Should we use TimeSpan.Zero there too?
Just curiosity.